### ISSUE TITLE
delivery person that was here today

### ISSUE THREAD

**Issue by ckimble2252** [LasershipSucks/LS-Use-List#2]
Just reporting this to you about todays delivery.

I've been in my office / warehouse since 8 a.m. today.
So, yes, this dept is open.
NO, I DON'T "SIT" ON THE DOCK AREA. and wait for deliveries.
plus the door is "ALWAYS UNLOCKED", even when I'm not here.
he knows this, bcs I told him, whether he cared or heard it at all, the first time he was here.

So, your delivery person / company 
-came in and opened the main door and looked in....literally 30 sec look
-he didn't see anyone "on" the dock / delivery area.
-closed the door
-he could see the warehouse was OPEN.
-but, he took it up on himself to unload the 8 cs of copy paper and left them "outside" against the dock garage door.
-The dock garage is 4' off the ground from the outside.
I received a phone call from the Wood St girls that the copy paper was outside the garage door.
He told the girls that there was no one there.
He didn't walk in at all.....he only opened the door for a brief second.

For the last 3 times he's been here for a delivery........he's been on his cell phone and doesn't even speak to me.
He knows to set the orders INSIDE.  I've told him that......whether I am here or not.

OBVIOUSLY, IF I WAS OFF, THE ORDER WOULD BE SITTING OUT THERE ALL DAY.
I'm the only person that works in this dept.

Long story short, I got the order in.
I know staff is hard to find. I know he doesn't work for you, but he does deliver and I don't know if anyone else has complained or not, but I am.
Thank you
Connie Kimble
Manager of Purchasing
Facility Services and
Vehicle Fleet Maintenance
[ckimble@northwoodhealth.com](mailto:ckimble@northwoodhealth.com)
